scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -235,6 +236,17 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
   /** Name patterns for output. */
   def scalaJSOutputPatterns: Target[OutputPatterns] = T { OutputPatterns.Defaults }
 
+  override def prepareOffline(all: Flag): Command[Unit] = {
+    val tasks =
+      if (all.value) Seq(scalaJSToolsClasspath)
```

Comment:
`scalaJSToolsClasspath` are not very big and necessary to do anything useful with ScalaJS (except plain compilation), so maybe they could be downloaded even without requiring the `all` flag.
What do you think?

Reply:
Yeah, makes sense. But with the same argument, we should always fetch Zinc dependencies in ScalaModule. Current scaladoc says:

| @param all If `true` , fetches also sources, Ammonite and compiler dependencies.